src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -703,7 +685,28 @@ private JObject ApplyHostConfiguration()
             return hostConfigObject;
         }
 
-        private void InitializeWorkers(string language)
+        internal static JObject LoadHostConfig(string configFilePath)
+        {
+            JObject hostConfigObject;
+            try
+            {
+                string json = File.ReadAllText(configFilePath);
+                hostConfigObject = JObject.Parse(json);
+            }
+            catch (JsonException ex)
+            {
+                throw new FormatException($"Unable to parse host configuration file '{configFilePath}'.", ex);
```

Comment:
missed adding log?

Reply:
Intentionally simplified

Reply:
Does this error show up in our system logs?

Reply:
Yes